README.md

```diff
@@ -36,14 +36,13 @@ ______________________________________________________________________
 
 ###### \*Codecov is > 90%+ but build delays may show less
 
-----
+______________________________________________________________________
```

Comment:
Are these accidental merge changes?

Reply:
pre-commit-ci[bot] did this and I don't know why, see https://github.com/Lightning-AI/lightning/pull/13283/commits/34fe10a8da95531f82489f96fd209a55e3095c84